rdfs-improved/README.md

```diff
@@ -156,12 +156,37 @@ The actual fixing can be done in two ways:
   - Export the graph to a file
   - Format the file as Turtle (see above)
 - Using a tool that does updates in-memory (eg Jena `update`)
-  - Run the tool with the original file and the update query
+  - Run `update` with the original file and concatenated update queries
   - Pass the result through the Turtle formatter
   - Save it to a file
 
 The latter is slightly simpler, so we use that.
 
+We write one Update per issue, using a strict structure to allow comprehension and evolution:
```

Comment:
@Sveino @griddigit-ci The PR is a big mess of many files, but please read this and comment